lib/modules/feed_press/feed_press.php

```diff
@@ -0,0 +1,227 @@
+        if ( ! $feedname || ! $token = get_option( 'podlove_module_feed_press' )['feed_press_api_key'] )
+            return;
+
+        $curl = new Http\Curl();
+        $curl->request( 'https://api.feed.press/feeds/subscribers.json?key=58eaa1be7018c&token=' . $token . '&feed=' . $feedname, array(
+            'headers' => array(
+                'Content-type'  => 'application/json'
+            )
+        ) );
+        $response = $curl->get_response();
+
+         if ($curl->isSuccessful()) {
+            return json_decode( $response['body'] );
+        } else {
+            return FALSE;
+        }
+    }
+
+    /**
+     * Lists all Publisher Feeds analyzed by FeedPress
+     *
+     * Podlove and FeedPress Feeds are matched by their URLs.
+     *
+     * The FeedPress Objects are returned with these properties:
+     *     - (int)      id
+     *     - (string)   name
+     *     - (int)      subscribers
+     *     - (string)   url
+     *     - (string)   original_url
+     * 
+     * @return array [all Feeds (Objects) which are analyzed by FeedPress]
+     */
+    public static function fetch_feeds_analyzed_by_feed_press() {
+        if ( ! $token = get_option( 'podlove_module_feed_press' )['feed_press_api_key'] )
+            return;
+
+        if ( ! $feed_press_feeds = self::fetch_feed_press_account_info($token)->feeds )
+            return;
+
+        $publisher_feeds = self::get_subscription_urls();
+        $feeds_analyzed_by_feed_press = [];
+
+        foreach ($feed_press_feeds as $feed) {
+            if ( in_array($feed->url, $publisher_feeds) ) {
+                $feeds_analyzed_by_feed_press[$feed->name] = $feed;
+            } else {
+                continue;
+            } 
+        }
+
+        return $feeds_analyzed_by_feed_press;
+    }
+
+    /**
+     * Fetches URLs of all public feeds provided by the current Podlove instance
+     * 
+     * @return array 
+     */
+    private static function get_subscription_urls() {
+        $feeds = [];
+
+        foreach (\Podlove\Model\Feed::all() as $key => $feed) {
+            if ( !$feed->enable || !$feed->discoverable )
+                continue;
+
+            /**
+             * @todo Think about if this check is useful. Couldn't it be that the redirected URLs match the FeedPress ones?
+             */
+            if ( $feed->redirect_http_status == '0' ) {
+                $feeds[] = $feed->get_subscribe_url();        
+            } else {
+                $feeds[] = $feed->redirect_url;
+            }
+        }  
+
+        return $feeds;
+    }
+
+    public function register_settings() {
+        if ( ! self::is_module_settings_page() )
+            return;
+
+        if ( $this->get_module_option('feed_press_api_key') == "" ) {
+            $auth_url = "https://api.feed.press/login.json?key=58eaa1be7018c&callback=" . urlencode(get_site_url().'/wp-admin/admin.php?page=podlove_settings_modules_handle') . "&response_type=code";
```

Comment:
Better extract `https://api.feed.press/` in a class constant like `API_BASE_URL` since you need it in multiple places.